code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -34,19 +35,31 @@
 	set_gun_light(new /obj/item/flashlight/seclite(src))
 	return ..()
 
-/obj/item/gun/energy/e_gun/stun
-	name = "tactical energy gun"
-	desc = "Military issue energy gun, is able to fire stun rounds."
+/obj/item/gun/energy/e_gun/hades
+	name = "SL AL-655 'Hades' energy rifle"
+	desc = "The standard issue rifle of the Nanotrasen Security Forces. Most have been mothballed into storage following the ICW, and aren't often issued to low ranking security divisions."
```

Comment:
Slight edit to iron out some awkward rhythm:
"The standard issue rifle of Nanotrasen's Security Forces. Most have been put in long term storage following the ICW, and usually aren't issued to low ranking security divisions."